htdocs/core/lib/functions.lib.php

```diff
@@ -218,21 +218,27 @@ function isModEnabled($module)
 
 	// Fix special cases
 	$arrayconv = array(
-		'bank' => 'banque',
+		'adherent' =>	'member',
+		'banque' => 'bank',
 		'category' => 'categorie',
+		'commande' => 'order',
 		'contract' => 'contrat',
-		'project' => 'projet',
-		'delivery_note' => 'expedition'
+		'expedition' => 'delivery_note',
+		'facture' => 'invoice',
+		'projet' => 'project',
+		'propal' => 'propale',
 	);
 	if (!getDolGlobalString('MAIN_USE_NEW_SUPPLIERMOD')) {
 		$arrayconv['supplier_order'] = 'fournisseur';
 		$arrayconv['supplier_invoice'] = 'fournisseur';
 	}
+
+	$module_alt = $module;
 	if (!empty($arrayconv[$module])) {
-		$module = $arrayconv[$module];
+		$module_alt = $arrayconv[$module];
 	}
 
-	return !empty($conf->modules[$module]);
+	return !empty($conf->modules[$module]) || !empty($conf->modules[$module_alt]);
```

Comment:
Imagine entry of function is 'delivery_note'.
It seems old code result will differ from new code result.

Reply:
OK, I can see the table is inconsistent.
I added conversions as needed (e.g. 'facture' is still used while 'invoice' is the actual module name), considered that the 'old' names must be mappable to the new names, but not the inverse - but he old entries map new to old (while the old names should no longer be the active ones in the code).

So I change the remaining mappings and if anything fails, then I guess the code needs updates elsewhere.